File: web_screen/src/main/java/com/bbogush/web_screen/MySdpObserver.java
package com.bbogush.web_screen;

import android.util.Log;

import org.webrtc.SdpObserver;
import org.webrtc.SessionDescription;

class MySdpObserver implements SdpObserver {
    private final String tag = "WebRtcManager";

    MySdpObserver() {
    }

    @Override
    public void onCreateSuccess(SessionDescription sessionDescription) {
        Log.d(tag, "onCreateSuccess() called with");
//        Log.d(tag, "onCreateSuccess() called with: sessionDescription = [" + sessionDescription.description + "]");
    }

    @Override
    public void onSetSuccess() {
        Log.d(tag, "onSetSuccess() called");
    }

    @Override
    public void onCreateFailure(String s) {
        Log.d(tag, "onCreateFailure() called with: s = [" + s + "]");
    }

    @Override
    public void onSetFailure(String s) {
        Log.d(tag, "onSetFailure() called with: s = [" + s + "]");
    }
}

File: webrtc_lib/src/main/java/com/android/mylibrary/MySdpObserver.kt
package com.android.mylibrary

import org.webrtc.SdpObserver
import org.webrtc.SessionDescription

internal open class MySdpObserver : SdpObserver {

    override fun onCreateSuccess(sessionDescription: SessionDescription) {
        "onCreateSuccess() called with".logE()
//        "onCreateSuccess() called with: sessionDescription = [" + sessionDescription.description + "]".logE()
    }

    override fun onSetSuccess() {
        "onSetSuccess() called".logE()
    }

    override fun onCreateFailure(s: String) {
        "onCreateFailure() called with: s = [$s]".logE()
    }

    override fun onSetFailure(s: String) {
        "onSetFailure() called with: s = [$s]".logE()
    }
}
